### ISSUE TITLE
What is it?

### ISSUE THREAD

**Issue by Ma1hT3ach3r** [Vinaymeldrum/Scrambler-Android-App#8]

**Comment by VinMeld** [comment]
Hello Mr. Osudar! Just had a week free before classes begin, and have been working on an android app where you have a few seconds to unscramble a random word. # of correct words are checked and stored in a database, and later displayed. All for fun ^
Hope you're doing well!

**Comment by Ma1hT3ach3r** [comment]
Cool, reminded me of a cube scrambling apps that I've seen in cubing.js : 
https://github.com/cubing/cubing.js

Also if you or Michael want a free premium account at Codecademy, just respond with your email addresses(school or personal) and I'll add you to the the group I am creating.

No pressure though, I am sure you have plenty on the plate.

Keep up the good work!

**Comment by VinMeld** [comment]
Yeah that would be really great! My email is vinaymeldrum@gmail.com and Michael's michael.lam8@live.com
Thank you so much for thinking of us! Good luck with your cubing repo it looks interesting!

**Comment by Ma1hT3ach3r** [comment]
They are sent. Also, I'm always looking for current computer science/engineering/STEM to do presentations on topics... like coding challenges, or mini workshops(virtually, for a variety of reasons) to get students more aware Computer science/programming/coding/STEM...etc. so, if you are interested in something like that(i.e. putting together a presentation/workshop/coding challenge) let me know and I will find time to have you come in(virtually) to either ICS4U(during class) or during the coding club

**Comment by michaeljylam** [comment]
Hi Mr. Osudar,

Sorry for the late reply, we both don't get notifications here and we're working on this app sporadically, so we missed your message, but it's good to hear from you! Hope things are going well, and a big thanks for the Codecademy access!

We'd certainly be interested in putting something together! We were thinking of maybe doing something for coding a Discord bot, since we both have experience with that and the platform is quite widespread (particularly in university too!). Let us know if that works with you, or if you had something else specific in mind!

I also turned on notifications for this thread so I'll be seeing your messages, but if it's easier for you, feel free to email us instead!

**Comment by Ma1hT3ach3r** [comment]
Looks like I too missed your message... maybe we can aim for something in the new year. 
I think a discord bot would be a great tutorial! The only problem is that OCDSB has "banned" discord. Maybe making a twitter bot? I know that Dan Shiffman(coding train) has a tutorial on this, and twitter is accessible to everyone!
I've seen student in my class make one in my first year teaching ICS4U, so I know it can be done in a reasonable amount of time and access to just the tools they need.

We haven't got a coding club running yet, as I've been trying to get a Debate Club going because there has been a fair amount of interest in it. I would be open to have you two run it as me, and possibly other educators, sponsor it.

I was actually just approached yesterday about it from a former student. We would run it using google meet and I'd get you the invite to the meetings... ocdsb isn't the most friendly(in terms of set up) for having guests in the classroom but if I know I've got to enthusiastic leaders(who pop in to guide students in tutorials) then I don't mind supporting it. Would be once a week, and we can pick a day that works for the two of you.

Just thoughts at this point! Hope all is well!
